src/test/Java/AnagramKataTest.java

```diff
@@ -0,0 +1,53 @@
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+
+public class AnagramKataTest {
+    AnagramKata anagramKata;
+@Before
+public void setUp() throws FileNotFoundException {
+ anagramKata = new AnagramKata("wordlist.txt");
+}
+
+    @Test
+    public void txtFileWordsWithNoDuplicates_AreSplitIntoSets() throws FileNotFoundException {
```

Comment:
Now that the implementation is using maps rather than sets, are these tests still relevant, if not then remove them.